Add meta descriptions (#4003)
wazuh/wazuh-documentation: source/user-manual/agents/listing/wazuh-app.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: You can list and see basic information about all registered agents by going to the Agents tab in the Wazuh app.

Reviewer: ```suggestion
  :description: With Wazuh, you can list and see basic information about all registered agents by going to the Agents tab of the Wazuh Kibana plugin.
```
